run_benchmarks.rb

```diff
@@ -57,13 +57,21 @@ def have_yjit?(ruby)
   ruby_version.downcase.include?("yjit")
 end
 
+# Disable Turbo Boost while running benchmarks. Maximize the CPU frequency.
 def set_bench_config(turbo:)
+  # sudo requires the flag '-S' in order to take input from stdin
   if File.exist?('/sys/devices/system/cpu/intel_pstate') # Intel
-    # sudo requires the flag '-S' in order to take input from stdin
-    check_call("sudo -S sh -c 'echo 1 > /sys/devices/system/cpu/intel_pstate/no_turbo'") unless turbo || intel_no_turbo?
+    unless intel_no_turbo? || turbo
+      check_call("sudo -S sh -c 'echo 1 > /sys/devices/system/cpu/intel_pstate/no_turbo'")
+      at_exit { check_call("sudo -S sh -c 'echo 0 > /sys/devices/system/cpu/intel_pstate/no_turbo'", quiet: true) }
```

Comment:
It seems better to not make it quiet here to let the user understand why e.g. the sudo password is asked again

Reply:
Sounds fair. Feel free to file a PR for it, but so far the 15-minute sudo timeout has been long enough for me to never encounter the situation when running benchmarks locally.